ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Make sure you are using the **latest** version of youtube-dl. Latest youtube-dl version at this moment is **2016.03.06**. Read [our FAQ](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) if you have troubles updating.
+- [ ] I've verified that I'm running youtube-dl version **2016.03.06**
+
+**What is the purpose of this issue? Put an `x` to the relevant box**
+- [ ] Site request (add support for a new website)
```

Comment:
Fixed. I was also thinking it might be better to have the example URL directly after the 'kind-of-issue' section. Now it's the latest section of the template, people might forget/miss it after all. Thoughts?